### PR TITLE
Explorer: Separate Node Sections

### PR DESCRIPTION
**Pull request by marwan-at-work**
Note that if there's only a single group, render it directly. 

Screenshot 2023-08-03 at 2 56 25 PM

Fixes ENG-1971

### CODE REVIEW

**Comment by samlinville** [comment]
Something is a little off here — it looks like these headings are implemented as just regular tree items, but there's some special kind of "header" item that we should use instead.
CleanShot 2023-08-03 at 16 16 32@2x

**Comment by samlinville** [comment]
Another example of the way that top-level My Nodes and All Nodes should be rendered: 
image

**Comment by samlinville** [comment]
It looks like the way that VS Code refers to these things is as "Views," so rather than one View that has My/All nodes separated inside the tree, they'd be two separate Views altogether.

For the sake of time, I'm fine if we implement this way for now, and fix it when we add Favorites.

File: tsrelay/handler/get_peers.go [hunk 1]
```diff
@@ -64,7 +69,12 @@ func (h *handler) getPeers(ctx context.Context, body io.Reader) (*getPeersRespon
 		return nil, err
 	}
 
-	s := getPeersResponse{Peers: make([]*peerStatus, 0, len(st.Peer))}
+	s := getPeersResponse{
+		PeerGroups: []*peerGroup{
+			{Name: "MY NODES"},
+			{Name: "ALL NODES"},
```
Comment (samlinville) [inline tsrelay/handler/get_peers.go:75]:
```suggestion
			{Name: "My nodes"},
			{Name: "All nodes"},
```

**Comment by tylersmalley** [comment]
@samlinville, as you pointed out, we would need to implement separate views to separate these. We absolutely should do that, but there is some more plumbing we need to put in place to have a single data pipeline so we don't duplicate call calls.

**Review by tylersmalley — APPROVED** [review]

**Pull request merged**